CaT/src/Filter/FilterFactory.php

```diff
@@ -163,6 +163,37 @@ public function dateperiod_overlaps_predicate($field_start, $field_end) {
 		};
 	}
 
+	/**
+	 * Map this over a dateperiod to get the standard behaviour of
+	 * overlapping periods using Field varables.
+	 *
+	 * @param	Predicates\Field	$field_start
+	 * @param	Predicates\Field	$field_end
+	 * @return	\Closure
+	 */
+	public function dateperiod_overlaps_predicate_fields(Predicates\Field $field_start,Predicates\Field $field_end) {
```

Comment:
derived the name from dateperiod_overlaps_predicate and wanted to express, that fields instead of field-ids are used